presto-tpcds/src/main/resources/META-INF.services/com.facebook.presto.spi.Plugin

```diff
@@ -0,0 +1 @@
+com.facebook.presto.tpcds.TpcdsPlugin
```

Comment:
Interesting. Can you explain how that works? It seems like the tpch connector has this file and also packages the module as a presto-plugin.